public/src/css/style.css

```diff
@@ -1930,6 +1930,15 @@ ol.search_list > li::before {
     height: 50px;
 }
 
+.capitalize-first-letter {
+    /*needed for :first-letter to work*/
+    display: inline-block;
+}
+
+.capitalize-first-letter:first-letter {
```

Comment:
This means we can write 'very big' as 'Very big' (and not 'Very Big' which `text-transform: capitalize` would do if left to its own devices).

Reply:
(Stakeholder request)